### DISCUSSION TITLE
shortcut for value.to(unit).magnitude

### DISCUSSION THREAD

**Discussion by mhechthz** [hgrecco/pint#1901] [Ideas]
I built me followig shortcut:

```
class myQuantity(pint.Quantity):
    def mag(self, unit):
        return self.to(unit).magnitude
class myUnit(pint.UnitRegistry):
    Quantity = myQuantity 

Unit = myUnit()

value = 1.0*unit.kg
print(value.mag('tonne'))
# instead of
print(value.to('tonne').magnitude)

```
This might be a nice feature for the standard library, since often one (me) needs the magnitude in the right unit.

**Comment by dimapu** [comment]
There's `m_as()` method which does this: https://pint.readthedocs.io/en/stable/api/base.html?highlight=m_as#pint.Quantity.m_as

**Reply by mhechthz** [reply]
Well, fine. Ever when I have a good idea, someone had it before ... 

Good work! Thank You!

**Reply by hgrecco** [reply]
Great minds think alike :-P
